### PR TITLE
add type hints for issues.py and types.py

### PR DESCRIPTION
**Pull request by RaileyHartheim**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Issue Number: N/A

### CODE REVIEW

**Review by rekby — COMMENTED** [review]
Great!

Thanks for your work. I have only cosmetic suggestions.

File: ydb/issues.py [hunk 1]
```diff
@@ -43,10 +47,26 @@ class StatusCode(enum.IntEnum):
...
+        self.issue_code = issue_code
+        self.severity = severity
+        self.issues = issues
+
+
 class Error(Exception):
     status = None
 
-    def __init__(self, message, issues=None):
+    def __init__(self, message: str, issues=None):
```
[the remark this answers is not in the document]
Comment (RaileyHartheim) [reply ydb/issues.py]: OK, I fixed it.

File: ydb/types.py [hunk 2]
```diff
@@ -4,73 +4,101 @@
 import json
 from . import _utilities, _apis
 from datetime import date, datetime, timedelta
+import typing
 import uuid
 import struct
 from google.protobuf import struct_pb2
 
+from .table import TableClientSettings
```
[the remark this answers is not in the document]
Comment (RaileyHartheim) [reply ydb/types.py]: All TableClientSettings imports are fixed.

File: ydb/issues.py [hunk 3]
```diff
@@ -43,10 +47,30 @@ class StatusCode(enum.IntEnum):
...
+
+
 class Error(Exception):
     status = None
 
-    def __init__(self, message, issues=None):
+    def __init__(
+            self,
+            message: str,
+            issues: typing.Optional[typing.Interable[_IssueMessage]] = None
```
Comment (rekby) [inline ydb/issues.py:72]: typo:
Interable 
-> 
Iterable

**Review by rekby — APPROVED** [review]

**Comment by rekby** [comment]
@RaileyHartheim, thanks for the work.

Type hints is great for understand relations in the code.

**Pull request merged**